### ISSUE TITLE
docker/jetson-nano/Dockerfile.python build failure

### ISSUE THREAD

**Issue by yury-sannikov** [roflcoopter/viseron#307]
Hi @roflcoopter , thank you for the great work you're doing!

I'm trying to build a jetson-nano docker image on Jetson Nano.
I'm getting this error message
```
 ---> Running in c15c74409958
gpg: directory '/root/.gnupg' created
gpg: keybox '/root/.gnupg/pubring.kbx' created
gpg: keyserver receive failed: No name
The command '/bin/sh -c gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys "$GPG_KEY"' returned a non-zero code: 2
```
It seems, `ha.pool.sks-keyservers.net` is not supported anymore. Replacing it with the `keyserver.ubuntu.com` fixes the issue.

**Comment by roflcoopter** [comment]
Thanks for the heads up! Will fix it in #306

**Comment by roflcoopter** [comment]
Closed in v2
